packages/deprecation-crawler/src/lib/commands/history.command.ts

```diff
@@ -66,15 +67,24 @@ export const historyCommand: YargsCommandObject = {
         const initial = {
           version: tag,
           tag: tag,
+          // are used to only log new deprecations
+          // will be overwritten after crawl
+          deprecations: deprecations,
         } as CrawledRelease;
         const release = (await run(tasks, config)(initial)) as CrawledRelease;
         deprecations.push(...release.deprecations);
       }
 
       await git.checkout(current);
 
+      const sortedDeprecations = semverSort(
```

Comment:
Sort the deprecations based by semver conventions to correctly add the introduction version to the deprecation.